chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CacheInitializor.java

```diff
@@ -0,0 +1,21 @@
+package gov.healthit.chpl.domain;
```

Comment:
Not really, I was hoping you would look around and think of where might be appropriate. Two ideas: we are using the manager[.impl] packages as our service layer so that might be appropriate. Or you could make a new package under chpl (as a sibling of domain, manager, dto, etc) for caching stuff.